PubNub/Data/Service Objects/PNResult.m

```diff
@@ -103,7 +103,7 @@ - (instancetype)initForOperation:(PNOperationType)operation
         }
         // Received unknown response from service.
         else if (processedData && ![processedData isKindOfClass:NSDictionary.class]){
-            
+#warning should there be a log here?
```

Comment:
@jzucker2 this one used in _dictionaryRepresentation_ method, so if debug will be printed, this information will be there.
Normally this happens only if parser wasn't able to recognise received data. Used to report request error.